src/views/ContributorRewards/components/MonthlyClaim.tsx

```diff
@@ -16,12 +16,20 @@ const MonthlyClaim: React.FC = () => {
     return <Button disabled full text='Claim' variant='secondary' />
   }, [status, onClaimRewards, claimableQuantity])
 
+  const [currentMonth, setCurrentMonth] = useState('')
```

Comment:
seems like you could replace this with by passing down `setMonth` and `rewardsMonth` from `useRewards()`